_posts/2016-11-13-recurse-center-week-7.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "Week[6] @ The Recurse Center 🏁"
+comments: true
+categories: tech recurse
+---
+
+A week with the Winter 1s complete! Time for another review.
+
+### Overview
+
+I've started feeling much more focused at RC. The change in people has really helped this. Exited for what the next 5 weeks brings!
+
+### Emotional 😊
+
+#### **Blogging is causing me grief...**
+
+Once again getting deep into a new week and freaking out about getting a blog post done.
+
+Is it going to be an accurate portrayal of what I did last week? How was I feeling? What did I do? How do I summarize all that?
+
+It's hard to blog in the present about how I felt in the past... at least without it feeling forced and unnatural. My new approach is going to be blogging about my current feels and relating those feels to the past week.
+
+#### **So much comfort**
+
+This past week has felt very comfortable.
+
+There's a group of people here who I've known for the past 7 weeks.
+There's another group of people who are a blank slate. I can be whoever I want to be around them. They have no expectations for the space.
+
+As time goes on, I gain more familiarity with the environment and people around me. When I first got to New York everything was new and I struggled to take it all in. Now that I've adjusted I can do a better job focusing on different interests.
+
+#### **Election feels**
+
+Last week was the election and a large number of people at RC were very effected by the results.
+
+I was also shocked, but not being a US resident has buffered me from the full feels. Was unsure how to best support my fellow RCers, and still not sure.
```

Comment:
I think for a lot of us, just not mentioning it is enough--having it be everywhere all the time is a lot. I love that I can escape when I'm at RC :)